dcat/index.html

```diff
@@ -264,24 +264,20 @@ <h3>DCAT scope</h3>
         A <b>dataset</b> in DCAT is defined as a "collection of data, published or curated by a single agent, and available for access or download in one or more serializations or formats".
         A dataset is a conceptual entity, and can be represented by one or more <b>distributions</b> that serialize the dataset for transfer.
         Distributions of a dataset can be provided via <b>data services</b>.
-        Detailed properties for a data service API are out of the scope of this version of DCAT.
```

Comment:
This is a helpful clarification and does no harm. It should stay.

Reply:
I think it's confusing, because it suggest we haven't addressed the use of DCAT to describe APIs, though we have done that. We define a data service, and in doing so we provide detailed properties.